### ISSUE TITLE
ListView Listener randomly working in SlidingUpListView 

### ISSUE THREAD

**Issue by solorchid** [ksoichiro/Android-ObservableScrollView#124]
I'm using this SlidingUpListViewActivity and I found that when the toolbar is in the middle position(SLIDING_STATE_MIDDLE). The ListView's listener is not working well. It hardly response when I click on the ListView items. but when i slide the toolbar to the top (SLIDING_STATE_UP), everything back to normal, the listview responding is fast and great. I don't know why. does it happen on all devices or it's just me?

**Comment by ksoichiro** [comment]
Thanks for your feedback.

I couldn't reproduce the problem on my device (Android 5.1).
Could you give me some information?
(OS version, logs, or something.)

If you have changed some codes from SlidingUpListViewActivity, I'd like to know that too.

**Comment by solorchid** [comment]
Hi, Soichiro

Thanks for your reply, the thing is really weird actually.  I'm running your original code from GitHub (1.3.0-debug) on my Nexus 6 (Android 5.1).

my operation order is as shown in the following pictures.
 
![screenshot_2015-05-13-14-27-56](https://cloud.githubusercontent.com/assets/7596158/7618786/8fa281e4-f982-11e4-8686-ff5695ae7b9b.png)

![screenshot_2015-05-13-14-28-21](https://cloud.githubusercontent.com/assets/7596158/7618790/94b2bc80-f982-11e4-9e93-364f107f538f.png)

![screenshot_2015-05-13-14-28-34](https://cloud.githubusercontent.com/assets/7596158/7618795/9855e768-f982-11e4-81b3-9a32b64e7089.png)

![screenshot_2015-05-13-14-28-53](https://cloud.githubusercontent.com/assets/7596158/7618798/9bf8f284-f982-11e4-9b2b-3c719423d03d.png)



1. I firstly swipe the slider from bottom to the middle like in the FIG.1.  At this state, I click the list from item 1 to item 7 (totally 7 items) in a speed of two items/ per second. From the touch feedback ripple affect as well as the logcat, only two of them are triggered.   I repeat the same test with same touch speed, only two or four items are triggered (randomly). most of the times items are not triggered at all, no touch feedback no log. just feels like they r not clicked.

2. Then I keep swiping header from FIG.1 state to FIG 2 state. Note the header is not fully swiped to the top. Then i repeated the clicking test in the same speed (2 items per second). the result is exactly the same as in the 1. 40% to 50% of all the clicked Items are triggered and feedback. 

3. Then I fully swiped the header to the top as shown in FIG3 and repeated the same click test (2 items/ per second),  Responding is perfect and all the clicked items triggered and feedback immediately. 

4. So I swipe back the header from FIG.3 state to the FIG.4 state and repeated the same click test. responding is randomly again.


so for summary, the list view click even only perfectly running when header is fully swiped to the top. On other state( middle state or bottom state) the listview click even is not fully detected or even not detected at all.  So I think is it possibly caused by the interceptor listener or ur other scrollview callback that when the listview is not fully swiped, the listview listener is interrupted by the scroll listener or something like that? I haven't got time to invest in your code. But I'll feedback when I found something new.

Once again, ur work is amazing!!!!! thanks a lot!!!!

**Comment by ksoichiro** [comment]
Thanks for the information.

I found that if I drag (even if it's just 1px) while tapping the item, click event does not happen.

Perhaps I might have to add some play before dispatching the touch events.

I'll check further into it later.

**Comment by solorchid** [comment]
Yeah,  maybe that's the problem, it might think I'm dragging when I'm actually clicking, the interesting thing is I added an elevation to the list view for 4dp. It works much better now. At least it responses every of my click, but still a bit slow
